tests/sentry/db/test_silo_models.py

```diff
@@ -157,6 +161,10 @@ def test_silo_foreign_keys():
     validate_no_cross_silo_foreign_keys(fk_emeptions)
 
 
+def test_cross_silo_deletions():
+    validate_no_cross_silo_deletions(fk_emeptions)
```

Comment:
This validates that no existing deletion cascade configuration will break in hybrid cloud.  Important because we didn't have any check for this (some of these deletions are NOT foreign keys).